packages/core/src/ui-helpers.ts

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable import/no-unused-modules */
 export {
   StoredDeploymentDeserializer,
   StoredDeploymentSerializer,
 } from "./new-api/stored-deployment-serializer";
 export * from "./new-api/type-guards";
 export * from "./new-api/types/module";
 export * from "./new-api/types/serialized-deployment";
+export { formatSolidityParameter } from "./new-api/internal/formatters";
```

Comment:
Just so I am clear on the reasoning.

Because this will inevitable be used in both the UI project and the cli, we move it to core and leverage it from there; as the alternative is duplication?